client/src/app/appCtrl.js

```diff
@@ -221,19 +231,17 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
       worker: false, // multithreaded, !but does NOT work with other libs in app.js or streaming
       comments: "#",
       chunk: function(chunk, parser) {
-        if (!firstChunkComplete) {
-          streamParser = parser;
```

Comment:
The reason I was putting this here was to avoid assigning this over and over again. I don't know how expensive it is to keep assigning the same thing over and over though - probably less than doing the check for `firstChunkComplete`? If you want to remove `firstChunkComplete` you can. Note that it is initialized above.